### PR TITLE
fcitx5-*: update

### PR DESCRIPTION
**Pull request by Vonfry**
###### Motivation for this change
Update to new version.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/inputmethods/fcitx5/fcitx5-chinese-addons.nix [hunk 1]
```diff
@@ -53,7 +53,7 @@ mkDerivation rec {
 
   prePatch = ''
     ln -s ${pyStroke} modules/pinyinhelper/$(stripHash ${pyStroke})
-    ln -s ${pyTable} modules/pinyinhelper/$(stripHash ${pyTable})
+    ln -s ${pyTable}  modules/pinyinhelper/$(stripHash ${pyTable})
```
Comment (poscat0x04) [inline pkgs/tools/inputmethods/fcitx5/fcitx5-chinese-addons.nix:56]: Why add a space?

**Review by poscat0x04 — APPROVED** [review]
Looks good to me, though I have not tried to build them myself.

**Comment by bobby285271** [comment]
@ofborg build fcitx5-chinese-addons

**Comment by bobby285271** [comment]
Seems that [fcitx5-chinese-addons `5.0.9`](https://github.com/fcitx/fcitx5-chinese-addons/releases/tag/5.0.9) was re-tagged (https://github.com/fcitx/fcitx5-chinese-addons/commit/38c08cbabcef050afa842dcd2364cf79c7b6edd0 -> https://github.com/fcitx/fcitx5-chinese-addons/commit/147c90a927ff1877fcb6ac1c625108e50fc9e3d8), can you check the hashes?

**Comment by Vonfry** [comment]
@bobby285271 thank for your caution. It is fixed and pushed forcibly.

**Review by bobby285271 — APPROVED** [review]
Result of `nixpkgs-review pr 148803` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt512.fcitx5-qt</li>
    <li>libsForQt514.fcitx5-qt</li>
  </ul>
</details>
<details>
  <summary>13 packages built:</summary>
  <ul>
    <li>fcitx5</li>
    <li>fcitx5-chinese-addons</li>
    <li>fcitx5-configtool</li>
    <li>fcitx5-gtk</li>
    <li>fcitx5-lua</li>
    <li>fcitx5-m17n</li>
    <li>fcitx5-mozc</li>
    <li>fcitx5-rime</li>
    <li>fcitx5-table-extra</li>
    <li>fcitx5-table-other</li>
    <li>fcitx5-with-addons</li>
    <li>libime</li>
    <li>libsForQt5.fcitx5-qt (libsForQt515.fcitx5-qt ,plasma5Packages.fcitx5-qt)</li>
  </ul>
</details>

https://groups.google.com/g/fcitx-announce/c/PFb5-aU3cMU

Quick review looks good, tested basic Chinese input.

**Comment by bobby285271** [comment]
Thanks.

**Pull request merged**